docs/security.rst

```diff
@@ -310,3 +310,20 @@ standard port 443, you'll need to configure that too. Be aware that super user p
     # Optionally, set the server to listen on the standard SSL port.
     web_server_port = 443
     base_url = http://<hostname or IP>:443
+
+Impersonation
+'''''''''''''
+
+Airflow has the ability to impersonate a unix user while running task
+instances based on the task's ``run_as_user`` parameter, which takes a user's name.
+
+*NOTE* For impersonations to work, Airflow must be run with `sudo` as subtasks are run
```

Comment:
Sure, but here you want your sudoers file to limit where "sudo chown" is allowed to work (or even chroot just before). Ie something like this:

`airflow ALL= NOPASSWD:/bin/chown -R airflow\:user /var/lib/airflow/*`

This is why it so important to have well commented example "sudoers" file as it can be quite a challenge to set it up securely.